src/components/Donuts.js

```diff
@@ -1,32 +1,40 @@
 import { connect } from 'react-redux';
 import React, { Component } from 'react';
 import DonutList from './DonutList'
+import { Link } from 'react-router'
 import '../css/Donuts.css';
 
 class Donuts extends Component {
   render() {
     const { donuts } = this.props;
     if (!donuts || !donuts.length) {
       return (
-        <div>
-          <h1>We see that you're very, very, hungry.</h1>
+        <div className="Donuts Donuts-empty">
+          <h1>We see that you’re very, very, hungry.</h1>
           <p>
-            Thanks for stoppin' in.  Head over to the donut "Viewer" and start
-            making some tasty donuts!
+            Thanks for stoppin’ in. Head over to
+            {' '}
+            <Link
+              className="Donuts-link"
+              to="/view"
+            >
+              the donut “Viewer”
+            </Link>
+            {' '}
+            and start making some tasty donuts!
           </p>
           <span
             aria-label="Donut"
             role="img"
-            style={{fontSize: '600%'}}
           >
             🍩
           </span>
         </div>
       )
     }
     return (
-      <div>
-        <h3 id='donut_list_header'>Your donuts, your highness...</h3>
+      <div className="Donuts Donuts-populated">
+        <h3>Your donuts, your highness…</h3>
```

Comment:
Should probably refactor styles into a class name or do some global `h1`, `h2`, `h3` styles. For later!